wordpress-mobile/WordPress-iOS
# Dark mode: Fix no results background color

Refs #12320

This updates the background color set in `configureTableViewColors(view: UIView?)` which will affect several screens, but most notably the no results view.

![image](https://user-images.githubusercontent.com/517257/63302508-167fd180-c292-11e9-9a9b-d01ddea111a5.png)

To test:
- open stats or reader
- note that the background color for the no results view is now black in dark mode
